.travis.yml

```diff
@@ -2,11 +2,19 @@ sudo: false
 
 language: java
 
+git:
+  quiet: true
+
 jdk:
   - openjdk8
 
 install:
-  - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -PTravis
+  - >
```

Comment:
Any reason for `>` character?

Reply:
Thanks for pointing it out; I actually meant to use `|` instead. Fixed now.

Both of those symbols denote a block scalar literal in YAML; only `|` keeps newlines as is, and `>` is _supposed_ to replace them with spaces. So theoretically `>` _should not_ have worked. But apparently Travis has its own way of parsing `.travis.yml`, which is not entirely consistent with YAML specification.

Either way, now everything's fine